### PR TITLE
[3.x] Update definitions of get_mouse_position methods

### PR DESCRIPTION
**Pull request by madmiraal**
3.x version of #57385.

### CODE REVIEW

File: doc/classes/CanvasItem.xml [hunk 1]
```diff
@@ -302,7 +302,7 @@
 		<method name="get_local_mouse_position" qualifiers="const">
 			<return type="Vector2" />
 			<description>
-				Returns the mouse position relative to this item's position.
+				Returns the mouse's position reltative to the origin of this [CanvasItem].
```
Comment (akien-mga) [inline doc/classes/CanvasItem.xml:305]:
```suggestion
				Returns the mouse's position relative to the origin of this [CanvasItem].
```

File: doc/classes/InputEventMouse.xml [hunk 2]
```diff
@@ -16,10 +16,12 @@
 			The mouse button mask identifier, one of or a bitwise combination of the [enum ButtonList] button masks.
 		</member>
 		<member name="global_position" type="Vector2" setter="set_global_position" getter="get_global_position" default="Vector2( 0, 0 )">
-			The global mouse position relative to the current [Viewport]. If used in [method Control._gui_input] and if the current [Control] is not under the mouse, moving it will not update this value.
+			When received in [method Node._input] or [method Node._unhandled_input], returns the mouse's positon relative to the origin of the root [Viewport].
```
Comment (akien-mga) [inline doc/classes/InputEventMouse.xml:19]:
```suggestion
			When received in [method Node._input] or [method Node._unhandled_input], returns the mouse's position relative to the origin of the root [Viewport].
```

File: doc/classes/InputEventMouse.xml [hunk 3]
```diff
@@ -16,10 +16,12 @@
...
+			When received in [method Control._gui_input], returns the mouse's position relative to the origin of the [CanvasLayer] that the [Control] is in.
 		</member>
 		<member name="position" type="Vector2" setter="set_position" getter="get_position" default="Vector2( 0, 0 )">
-			The local mouse position relative to the [Viewport]. If used in [method Control._gui_input], the position is relative to the current [Control] which is under the mouse. If the current [Control] is not under the mouse, moving it will not update this value.
+			When received in [method Node._input] or [method Node._unhandled_input], returns the mouse's positon relative to the origin of the [Viewport] this [Node] is in.
```
Comment (akien-mga) [inline doc/classes/InputEventMouse.xml:23]:
```suggestion
			When received in [method Node._input] or [method Node._unhandled_input], returns the mouse's position relative to the origin of the [Viewport] this [Node] is in.
```

**Review by akien-mga — APPROVED** [review]

**Comment by madmiraal** [comment]
Updated to fix typos and includes changes agreed in the master branch version #57385.

File: doc/classes/CanvasItem.xml (hunk 1 above)
Comment (madmiraal) [reply doc/classes/CanvasItem.xml]: Changed to reflect changes agreed in the master branch version.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**